platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -611,17 +645,8 @@ public VirtualLeafRecord<K, V> loadLeafRecord(final K key) throws IOException {
         VirtualLeafRecord<K, V> leafRecord = pathToKeyValue.get(path);
 
         // FUTURE WORK: once the reconnect key leak bug is fixed, this block should be removed
```

Comment:
I assume the comment may be deleted, too

Reply:
Removed